pyproject.toml

```diff
@@ -82,6 +82,7 @@ doc = [
     "sphinx_autodoc_typehints",
     "docutils==0.16",  # Needed to show bullet points in sphinx_rtd_theme
     "nbsphinx-link",
+    "importlib_resources",
```

Comment:
Looks good to me. Thanks @msricher 

Just one question; should we have ``importlib_resources" as package dependence since it also shows in some tests dependency?
https://github.com/theochem/AtomDB/blob/8104303f837b09254fb34eaae48d4b687f7dc8a8/pyproject.toml#L72

Reply:
Yes, you are right! I will fix it.

Reply:
Actually it's already in `test_extra`.